src/index.html

```diff
@@ -6,9 +6,35 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a class="logo_img"
+         href="index.html">
+        <img src="images/logo.png" alt="MOYO">
+      </a>
```

Comment:
apply code styling recommendations from this [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md)
```suggestion
      <a 
        class="logo_img"
        href="index.html"
      >
        <img src="images/logo.png" alt="MOYO">
      </a>
```